pkg/ansible/runner/runner.go

```diff
@@ -52,62 +52,65 @@ type Runner interface {
 	GetFinalizer() (string, bool)
 }
 
+func ansibleVerbosityString(verbosity int) string {
+	if verbosity > 0 {
+		return fmt.Sprintf("-%v", strings.Repeat("v", verbosity))
+	}
+	return ""
+}
+
+type cmdFuncType func(ident, inputDirPath string, maxArtifacts int) *exec.Cmd
+
+func playbookCmdFunc(verbosity, path string) cmdFuncType {
+	return func(ident, inputDirPath string, maxArtifacts int) *exec.Cmd {
```

Comment:
It may be worth creating an issue to make it possible to configure the `ansibleVerbosity` via an annotation on the Custom Resource. Thoughts?

Reply:
I like that idea a lot actually

Reply:
https://github.com/operator-framework/operator-sdk/issues/2091